packages/cyberstorm/src/components/PackageFlag.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/PackageFlag.module.css";
+
+export interface PackageFlagProps {
+  label?: string;
+  icon?: ReactNode;
+  packageFlagStyle?: string;
+}
+
+/**
+ * Cyberstorm PackageFlag component
+ */
+export const PackageFlag: React.FC<PackageFlagProps> = (props) => {
+  const { label, icon } = props;
+  styles;
+  return (
+    <button type="button" className={"packageFlag"}>
```

Comment:
Why is the root element a button when clicking it does nothing?

Reply:
`className` should refer to a value from the imported styles, e.g. `className={styles.packageFlag}`. This also means you can remove the line 15. The way the actual rendered class attribute will include a hash that prevents collisions with class names from other components.

Reply:
It was decided that the className for the top level element in Cyberstorm components should be called `root` (instead of `packageFlag` as it's called here).

Reply:
> Why is the root element a button when clicking it does nothing?

Valid question. I don't see reason why it actually should be button so I'll fix that!